views/partials/dashboards/remove-form.hbs

```diff
@@ -144,7 +144,7 @@
         </div>
         {{/if}}
       </div>
-      {{> dashboards/remove-form-confirm}}
+      {{> dashboards/remove-form-confirm acctInfo=acctInfo}}
```

Comment:
Q: Above (L3) this was `this.acctInfo`. Should this be `acctInfo=this.acctInfo`?

Reply:
I think the `this` is not needed here. Honestly, we could probably get rid of it in a lot of places. Inside of a loop or hbs helper function, `this` gives context to the loop / function, but otherwise, at top level, it can be omitted.